lib/util.js

```diff
@@ -304,7 +304,12 @@ function formatValue(ctx, value, recurseTimes) {
       return ctx.stylize(RegExp.prototype.toString.call(value), 'regexp');
     }
     if (isDate(value)) {
-      return ctx.stylize(Date.prototype.toISOString.call(value), 'date');
+      if (value.toString() === 'Invalid Date') {
+        return ctx.stylize('Invalid Date', 'date');
+      }
+      else {
```

Comment:
Does this pass `make lint`?  The `else` should go on the previous line with the closing curly brace. Actually, core style is to leave off the braces all together in cases like this.